userspace/libsinsp/parsers.cpp

```diff
@@ -1448,6 +1457,34 @@ void sinsp_parser::parse_clone_exit(sinsp_evt *evt)
 	return;
 }
 
+void sinsp_parser::parse_execve_enter(sinsp_evt *evt)
+{
+	sinsp_evt_param *parinfo;
+	char fullpath[SCAP_MAX_PATH_SIZE];
+	uint16_t etype = evt->get_type();
+
+	switch(etype)
+	{
+		case PPME_SYSCALL_EXECVE_8_E:
+		case PPME_SYSCALL_EXECVE_13_E:
+		case PPME_SYSCALL_EXECVE_14_E:
+		case PPME_SYSCALL_EXECVE_15_E:
+		case PPME_SYSCALL_EXECVE_16_E:
+		case PPME_SYSCALL_EXECVE_17_E:
+			break;
+		case PPME_SYSCALL_EXECVE_18_E:
+			// Get the filename
+			parinfo = evt->get_param(0);
+			sinsp_utils::concatenate_paths(fullpath, SCAP_MAX_PATH_SIZE,
+										   evt->m_tinfo->m_cwd.c_str(), (uint32_t)evt->m_tinfo->m_cwd.size(),
```

Comment:
What indentation setting are you using? Does this line look good in your environment? I use 8 spaces for tabs and the indentation looks a bit messy. I understand we don't have an official convention on the coding guidelines, but you can see how it looks also on github.